parsec/core/types/backend_address.py

```diff
@@ -78,7 +84,24 @@ def from_url(cls, url: str):
         else:
             params = {}
 
-        path = unquote_plus(split.path)
+        if allow_http_redirection and split.scheme in ("http", "https"):
+            # `no_ssl` is defined by http/https scheme and shouldn't be
+            # overwritten by the query part of the url
+            params["no_ssl"] = ["true" if split.scheme == "http" else "false"]
+            # Remove the `/redirect/` path prefix
+            split_path = path.split("/", 2)
+            if split_path[:2] != ["", "redirect"]:
+                raise ValueError("HTTP to Parsec redirection URL must have a `/redirect/...` path")
+            try:
+                path = f"/{split_path[2]}"
+            except IndexError:
+                path = ""
```

Comment:
Isn't this a `ValueError` similar to the ones raised below?

Reply:
yup, the idea is `ValueError` is the standard exception if the provided URL doesn't respect the expected format